### PR TITLE
Migrated Batch 5 of Density Data

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: src/data/density-constants.ts [hunk 1]
```diff
@@ -780,6 +780,786 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
...
+    minimumTemperature: 189.79,
+    maximumTemperature: 461.15,
+    densityAtMinimumTemperature: 58.861,
+    densityAtMaximumTemperature: 14.493,
+  },
+
+  HydrogenSulfide: {
+    name: 'Hydrogen sulfide',
+    formula: 'H2S',
+    CAS: '04-06-7783',
```
Comment (saikat-samanta) [inline src/data/density-constants.ts:1206]: 7783-06-4

File: src/data/density-constants.ts [hunk 2]
```diff
@@ -780,6 +780,786 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
...
+    C3: 512.5,
+    C4: 0.24713,
+    minimumTemperature: 175.47,
+    maximumTemperature: 512.5,
+    densityAtMinimumTemperature: 27.915,
+    densityAtMaximumTemperature: 8.5942,
+  },
+
+  NMethylAcetamide: {
+    name: 'N-MethylAcetamide',
```
Comment (saikat-samanta) [inline src/data/density-constants.ts:1309]: N-Methyl acetamide

**Review by saikat-samanta — APPROVED** [review]
Rest good 👍

**Review by maneetgoyal — APPROVED** [review]
👍

**Pull request merged**